packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -149,8 +150,31 @@ function createInstanceMetadataServiceFactory(
     .flat();
   return createServiceFactory({
     service: instanceMetadataServiceRef,
-    deps: {},
-    factory: async () => ({ getInstalledFeatures: () => installedFeatures }),
+    deps: {
+      httpRouter: coreServices.rootHttpRouter,
+      logger: coreServices.rootLogger,
+    },
+    factory: async ({ logger, httpRouter }) => {
+      const instanceMetadata = {
+        getInstalledFeatures: () => installedFeatures,
+      };
+
+      logger.info(
+        `Installed features on this instance: ${instanceMetadata
+          .getInstalledFeatures()
+          .join(', ')}`,
+      );
+
+      const router = Router();
+
+      router.get('/features/installed', (_, res) => {
+        res.json({ items: instanceMetadata.getInstalledFeatures() });
+      });
+
+      httpRouter.use('/.backstage/instanceMetadata/v1', router);
```

Comment:
Had the thought that maybe this should be just `metadata`, since the fact that we're calling an API kind of implies that we're talking to an instance anyway. I don't dislike the current naming either, just wonder if there are any other similar thoughts.